.ci/scripts/run-test.sh

```diff
@@ -8,6 +8,7 @@ echo $USER
 echo $USERNAME
 
 source $NVM_DIR/nvm.sh
-cd $DOCKER_BASE_DIR
 nvm use
+## Let's use install rather than ci to help with a faster execution
+npm install
```

Comment:
I think this could be dangerous because `npm install` will always use the latest version of deps which aren't fixed with their exact versions in `package.json`, thus causing the tests to fail if a dependency has breaking changes which haven't properly bumped `major` or if we had tests which relied on a particular buggy behaviour which has changed.

IMO we should still use `npm ci` even if that means longer build times. It also ensures that the code we're testing is the exact code we'll ship.

More details on the behaviour of `ci` vs. `install` here: https://lucasfcosta.com/2020/10/17/lockfile-guide.html

Reply:
e8776ae is the proposal:
1) In the CI, it relies in the first stage about `npm ci` since the docker container mounts the workspace as volume
2) Locally, it's possible to customise the behaviour with a new argument:

```bash
$ .ci/scripts/run-test-in-docker.sh install
$ .ci/scripts/run-test-in-docker.sh ci
```

Reply:
Looks great, thank you! I'm very happy that we're mounting the workspace as a volume and thus can reuse deps rather than reinstalling everything. Thanks for that.